pyaedt/edb_core/components.py

```diff
@@ -1509,9 +1509,15 @@ def create_pingroup_from_pins(self, pins, group_name=None):
             return pingroup
 
     @pyaedt_function_handler()
-    def delete_single_pin_rlc(self):
+    def delete_single_pin_rlc(self, deactivate_only=False):
+        # type: (bool) -> list
         """Delete all RLC components with a single pin.
 
+        Parameters
+        ----------
+        deactivate_only : bool, optional
+            Whether to delete the components or keep them and deactivate only. Default is ``False`` to delete them.
```

Comment:
```suggestion
            Whether to only deactivate components rather than delete them.
            The default is ``False``, which deletes the components.
```